### ISSUE TITLE
Out of date information on the WPF (XAML) Getting Started Pages

### ISSUE THREAD

**Issue by JR-Morgan** [oxyplot/docs#45]
The Getting Started Guide for [WPF (XAML)](https://github.com/oxyplot/docs/blob/d9b167d/getting-started/hello-wpf-xaml.rst) is out of date. The [example code](https://github.com/oxyplot/documentation-examples/tree/62d3ddb/HelloWorld/WpfApplication1) still refers to the old www.oxyplot.org/wpf url.
The difference between [WPF (XAML)](https://github.com/oxyplot/docs/blob/d9b167d/getting-started/hello-wpf-xaml.rst) and [WPF](https://github.com/oxyplot/docs/blob/master/getting-started/hello-wpf.rst) pages is unclear and a cause for confusion.

Could the WPF (XAML) be deleted as the WPF section replaces it as far as I can tell. At the very least an out of date disclaimer should be added to the page.

**Comment by VisualMelon** [comment]
Yes, they absolutely need restructing/rewriting prior to the 2.1 release now that the XAML stuff has been moved. I was meant to do that a couple of weeks ago, but didn't find the time.

The WPF (XAML) stuff isn't meant to be replaced, but it is no longer support. You will - in 2.1 - need a reference to `oxyplot-contrib` to use it.

**Comment by rs38** [comment]
I also just stuck in that outdated "getting started" and for WPF beginners, most of the XAML error are not clear to understand.
It should be deleted, as https://github.com/oxyplot/docs/blob/master/getting-started/hello-wpf.rst works fine.
